test/PowerShellEditorServices.Test.Shared/Completion/CompleteNamespace.cs

```diff
@@ -0,0 +1,30 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.PowerShell.EditorServices.Services.TextDocument;
+
+namespace Microsoft.PowerShell.EditorServices.Test.Shared.Completion
+{
+    internal class CompleteNamespace
+    {
+        public static readonly ScriptRegion SourceDetails =
+            new ScriptRegion(
+                file: TestUtilities.NormalizePath("Completion/CompletionExamples.psm1"),
+                text: string.Empty,
+                startLineNumber: 21,
+                startColumnNumber: 25,
```

Comment:
I think you want this to be like 15
```suggestion
                startColumnNumber: 15,
```